src/error.rs

```diff
@@ -96,14 +101,21 @@ impl StdError for Error {
         let e = match *self {
             Error::Custom(ref e) => e.as_ref(),
             Error::WrongGeneration(ref e) => e,
-
+            Error::NoMarker => unimplemented!(),
+            Error::NoEntity => unimplemented!(),
             Error::__NonExhaustive => unimplemented!(),
         };
 
         Some(e)
     }
 }
 
+impl PartialEq for Error {
```

Comment:
This is really hacky, but I want sure if I could impl PartialEq for BoxedErr.

Reply:
The `BoxedErr` variant should just always return `false`, just avoid implemeting `Eq`.